src/core/a-entity.js

```diff
@@ -539,7 +539,7 @@ var proto = Object.create(ANode.prototype, {
    */
   setAttribute: {
     value: function (attr, value, componentPropValue) {
-      var isDebugMode = this.sceneEl && this.sceneEl.getAttribute('debug');
+      var isDebugMode = this.sceneEl && this.sceneEl.hasAttribute('debug');
```

Comment:
The spec refers to boolean attributes, though A-Frame components have a little bit more juice. That spec always confused people (`disabled="false"` means disabled?) too, so if we can avoid that confusion, it would be nice.

I think @dmarcos is right, making `getAttribute('debug')` on `<a-scene debug>` return the default component value seems best because we've had to do this in other places: https://github.com/aframevr/aframe/pull/1474/files#diff-eb9bb1084863a2f65678ea67d6e7cf5eR197

Thanks for the work so far, I'll look into advice on how to do this.